README.rst

```diff
@@ -145,3 +147,11 @@ simply be appended to the invocation as follows::
     python3 -m pymongoexplain <path/to/your/script.py> [PARAMS] [--optname OPTS]
 
 
+Limitations
+-----------
+This package does not support the fluent `Cursor API <https://api.mongodb.com/python/current/api/pymongo/cursor.html>`_,
+so if you attempt to use it like so::
+
+    ExplainCollection(collection).find({}).sort(...)
+
+it will fail.
```

Comment:
Can you offer a suggested workaround? Something like: 
```
Instead pass all the arguments to the find() call, like so::

    ExplainCollection(collection).find({}, sort=...)

```

Reply:
Done.